### DISCUSSION TITLE
Please reply to my support tickets [Github Support Team]

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [community/community#111876] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

@davecheney @queenofcorgis @ettaboyle @stevepopovich

Sorry for the ping! I literally beg 🙏🏻 you all to please reply to my GitHub Support tickets: #2645708 and #2658272, or at least forward my message to the internal team to reply 🙏🏻 

Thanks in advance.

**Comment by tonmoyDeveloperBD** [comment]
Hi

**Comment by tonmoyDeveloperBD** [comment]
Hi

**Comment by Shahseo** [comment]
I understand that the GitHub support team is likely busy, but these tickets are quite urgent and require your attention. If possible, I would greatly appreciate a reply or an update on the status of my tickets.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @vedantmgoyal9 


Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
